README.md

```diff
@@ -87,29 +87,35 @@ Summary of one-time project setup:
 $ source venv/bin/activate
 ```
 
-2. Find the test file that contains the test you want to run.
+2. Check the `tests/unit_tests` folder, and find the test file you want to run
 
-   - Check the `tests` folder, and find the test file you want to run
    - In that test file, read through each test case
+      - If it is incomplete, complete the test. **There are incomplete tests for Waves 01, 02, and 06**. 
+        - *Is this a nominal or edge case?*
+        - *What type of input do we need to test this case?*
+        - *What is the expected output for the given input?*
+   - Remove the lines that contain `@pytest.mark.skip()` for the test(s) you want to run.
 
-3. Run the tests for your specific wave
+3. Run the test(s)! (RED)
```

Comment:
I meant to add this line to the top of the project workflow "We will use a Test Driven Development programming workflow to work on this project. Notice the Red-Green-Refactor steps in the workflow steps outlined below.", and then add (GREEN) to step 6

Reply:
I'm not familiar enough with our curriculum to know the answer to this, but will students have seen this notation before? I know a lot of folks will probably immediately get red is for test should be not passing here, but my instinct is this could raise confusion if they haven't definitely seen it before, even with the paragraph above.

Reply:
Yea, I think you're right that it adds more confusion than value. I'll remove it here and from the viewing party README.